### PR TITLE
BLT-1410 - BL expandable column

### PR DESCRIPTION
**Pull request by swifttarrow**
## Description
This PR makes a bunch of changes to our fixed-data-table fork:
* On hover, the knob indicator will span the full height of the column.
* The color of the knob indicator was changed from blue to orange (our brand primary color).
* The color of the divider line while resizing was changed from blue to orange.
* While resizing, an opacity was added to the table.
* The width of the knob indicator was decreased from 4px to 1 px..

## Test Description
Refer to the [FE PR](https://github.com/BuildingConnected/client/pull/7338).

## Merge Checklist
- [x] Bump version
- [x] Publish

### CODE REVIEW

File: src/FixedDataTableCell.react.js [hunk 1]
```diff
@@ -121,13 +121,18 @@ var FixedDataTableCell = createReactClass({
     var columnResizerComponent;
     if (props.onColumnResize) {
       var columnResizerStyle = {
-        height
+        height,
+        opacity: props.isHoveringResizerKnob ? 1 : 0,
       };
+      console.log('isHovering: ', props.isHoveringResizerKnob)
```
Comment (pete-bc) [inline src/FixedDataTableCell.react.js]: @swifttarrow looks like you forgot to remove these logs. 🤔 Do we have the linter turned off for this file extension? I'd expect it'd have caught this

**Review by pete-bc — DISMISSED** [review]
Noted some issues on the FE PR

File: src/FixedDataTableCell.react.js (hunk 1 above)
Comment (swifttarrow) [reply src/FixedDataTableCell.react.js]: @pete-bc Hrm, that's odd. Lemme fix; thanks for catching!

**Comment by swifttarrow** [comment]
@pete-bc Fixed! Thanks for reviewing; ready for another round.

**Review by pete-bc — APPROVED** [review]
lookin good

**Pull request merged**